dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -87,6 +91,9 @@ http.remoteip.proxies.desc=A regular expression that defines trusted proxies.
 http.remoteip.useRemoteIpInAccessLog=Use remote IP in access log
 http.remoteip.useRemoteIpInAccessLog.desc=If this property is set to true and the remote client information was verified by the HTTP Channel, the NCSA access log reflects the Forwarded or X-Forwarded-* headers. These headers are reflected when one or more of the following items are recorded: the remote client IP, the host, or the request protocol.
 
+http.compression=Compression Options
+http.compression.desc=An element configured within the httpEndpoint element so that the associated HTTP channel is enabled to consider compressing response bodies based on the client's Accept-Encoding header.
```

Comment:
**change**
An element configured within the httpEndpoint element so that the associated HTTP channel is enabled to consider compressing response bodies based on the client's Accept-Encoding header.
**to**
An element that is configured within the httpEndpoint element, so that the associated HTTP channel can consider compressing response bodies based on the client's Accept-Encoding header.